test/e2e/storage/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- TODO
```

Comment:
BTW, We should probably have a sig-storage [alias](/kubernetes/kubernetes/blob/master/OWNERS_ALIASES) that we could use in different places (including this one).